internal/lldp/lldp.go

```diff
@@ -74,54 +70,34 @@ func NewLLDP(opts *Options) *LLDP {
 	return &LLDP{
 		interfaceName:    opts.InterfaceName,
 		advertiseOptions: opts.AdvertiseOptions,
-		enableRx:         opts.EnableRx,
-		enableTx:         opts.EnableTx,
-		rxWaitGroup:      &sync.WaitGroup{},
-		l:                opts.Logger,
 		neighbors:        make(map[neighborCacheKey]Neighbor),
 		onChange:         opts.OnChange,
+		Rx:               &RunningState{},
```

Comment:
We don't care about the initial enabled state because we're going to call the same `SetRxAndTx` method as would be used when changing options. This gives a single path by which Rx/Tx is Start/Stop.

Also means we can eliminate the `Start()` method entirely